### ISSUE TITLE
ADC_Common_TypeDef structs do not match reference manuals

### ISSUE THREAD

**Issue by tcbennun** [STMicroelectronics/cmsis_device_h5#3]
This applies to all four chips, but taking H503 as an example:

[Definition in header](https://github.com/STMicroelectronics/cmsis_device_h5/blob/fc83d7330583da0ee111d87f3d96aa5c85a970fd/Include/stm32h503xx.h#L1071) defines CSR, CCR and CDR registers:

```.c
typedef struct
{
  __IO uint32_t CSR;          /*!< ADC common status register,            Address offset: 0x300 + 0x00 */
  uint32_t      RESERVED1;    /*!< Reserved,                              Address offset: 0x300 + 0x04 */
  __IO uint32_t CCR;          /*!< ADC common configuration register,     Address offset: 0x300 + 0x08 */
  __IO uint32_t CDR;          /*!< ADC common group regular data register Address offset: 0x300 + 0x0C */
} ADC_Common_TypeDef;
```

RM0492 meanwhile defines CCR, HWCFGR0, VERR, IPDR and SIDR registers (<s>and no CSR or CDR</s> _chips other than H503 have these, my mistake_):

<img width="1060" alt="image" src="https://github.com/STMicroelectronics/cmsis_device_h5/assets/40573959/722ba726-1622-4aee-9f15-a74c1e24a949">

**Comment by RJMSTM** [comment]
ST Internal Reference: 174587

**Comment by TOUNSTM** [comment]
fixed in 9a778e0f162bd21200647adafb922d50b2c60f81
